README.md

```diff
@@ -26,56 +28,71 @@ npm start
 
 Browse http://localhost:8080 and open the dev tools console to see the proof being generated and its verification
 
-
 ### Usage
 
 #### Initializing the library
+
 ```js
 import * as rln from "@waku/rln";
 
 const rlnInstance = wait rln.create();
 ```
 
 #### Generating RLN membership keypair
+
 ```js
 let memKeys = rlnInstance.generateMembershipKey();
 ```
 
+#### Generating RLN membership keypair using a seed
+
```

Comment:
I'd had a comment that the ideal seed size is 32 bytes, less reduce security, more is unnecessary entropy. Right, @s1fr0 ?